linera-chain/src/data_types.rs

```diff
@@ -180,6 +180,25 @@ impl IncomingBundle {
             (message_id, posted_message)
         })
```

Comment:
nit: Can we make `check_signatures` a method of `LiteValue`?

Reply:
I think it makes sense to allow for checking signature of any type of `CryptoHash` - not only `LiteValue`.